src/tools/vs-code.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Visual Studio Code
+description: You can use VS Code to develop Dart apps.
+toc: false
+---
+
+With the Dart plugin, you can use [Visual Studio Code (VS Code)][vs-code]
+to develop Dart apps.
+Here's where you can find more information:
+
+* [Set up an editor][setup] on the Flutter site has
```

Comment:
Maybe clarify that this also works for Dart command-line and server apps?

Reply:
You mean the Flutter instructions, or VS Code in general? I thought it worked for web apps, too. (Obviously, I haven't tried it.)

Reply:
VS Code works great for all kinds of Dart apps: command-line, server, web, and Flutter. But the current phrasing and link to Flutter can be mis-read to suggest it's for Flutter only.